tests/test_context.py

```diff
@@ -19,8 +19,9 @@ class TestContext(TestCase):
     _test_few_facets = 'project,model,index_node,data_node'
 
     def setUp(self):
-        self.test_service = 'http://esgf-data.dkrz.de/esg-search'
-        # self.test_service = 'https://esgf.ceda.ac.uk/esg-search'
+        # DKRZ have retired CMIP5 data
```

Comment:
Maybe they have removed data from this particular model? There is definitely still CMIP5 data there: https://esgf-data.dkrz.de/esg-search/search?format=application%2Fsolr%2Bjson&project=CMIP5 returns many results.

Reply:
eugh maybe they'll remove data from the model we replace it with, in CEDA we trust, man :grin:

Reply:
```suggestion
```
Switching to CEDA for the tests seems fine, but please remove the comment as it is misleading (at least for now)